tests/sanity/ignore-2.15.txt

```diff
@@ -0,0 +1,43 @@
+plugins/module_utils/turbo/exceptions.py compile-2.7!skip
```

Comment:
Not worth holding up this PR over, but we could probably remove most of this sanity file, and most of the 2.12+ sanity files, by adding a `tests/config.yml` file that requires python 3.6.